FreeTubeApp/FreeTube
# Set new consent cookie and supress tracking cookies on the watch page

# Set new consent cookie and supress tracking cookies on the watch page




## Pull Request Type


## Description
This pull request set the new consent cookie, based on what yt-dlp uses, although the old cookie still seems to be used too.

It also gets rid of tracking cookies that YouTube is returning, maybe we should consider changing the values of the consent cookies at some point, so that YouTube doesn't return those cookies.

## Screenshots 
before:
![before](https://github.com/FreeTubeApp/FreeTube/assets/48293849/e94d8842-5a33-4bca-8d9a-408f261ec312)

after:
![after](https://github.com/FreeTubeApp/FreeTube/assets/48293849/bd126054-4d7c-4a5b-a99a-53be2c9435a2)

## Testing 
1. Open a video
2. In the devtools look at the network log
3. Click on `https://www.youtube.com/sw.js_data` and `https://www.youtube.com/iframe_api`
4. Switch to their cookies tab
5. Check that only the `CONSENT` and `SOCS` cookies are listed
